day 2/exercise 2/solution.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"fmt"
+	"math/rand"
+	"sync"
+	"time"
+)
+
+var rate int
+
+func findrating(ind_rate int, wg *sync.WaitGroup) {
+
+	defer wg.Done()
+	//fmt.Println(ind_rate)
+	response_time := rand.Intn(5)
+	time.Sleep(time.Duration(response_time))
+	rate += ind_rate
+
+}
+func main() {
+	var wg sync.WaitGroup
+	nos := 200
+	rate = 0
+	for i := 1; i <= nos; i++ {
+		wg.Add(1)
+		ind_rate := rand.Intn(10)
+		go findrating(ind_rate, &wg)
```

Comment:
Why are you passing the ind_rate in the findrating function call? It is like `you are telling your classmates to give this rating to our class teacher`. Ideally, the findrating function should generate the ind_rate. Also, prefer camelCase.